test/e2e_node/container.go

```diff
@@ -134,10 +149,10 @@ func (cc *ConformanceContainer) Present() (bool, error) {
 	return false, err
 }
 
-type ContainerState uint32
+type ContainerState int
```

Comment:
@dchen1107 In fact this is not necessary. Before we used bit operations, now we do not, then I changed `uint32` to `int`.